robotiq_s_model_articulated_msgs/package.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<package>
+  <name>robotiq_s_model_articulated_msgs</name>
+  <version>1.0.0</version>
+  <description>Messages for the robotiq_s_model_articulated model.</description>
+
+  <maintainer email="dash@clearpathrobotics.com">Devon Ash</maintainer>
+  <license>BSD</license>
```

Comment:
I authored the package. However, the two messages were originally defined here https://bitbucket.org/osrf/drcsim/src/7aee32718c4a74419b2bb447ab41d8ab9e4edf68/atlas_msgs/?at=default as apart of a larger package; atlas_msgs.